packages/wonder-blocks-data/src/util/gql-types.js

```diff
@@ -14,43 +19,41 @@ export type GqlOperation<
     // correct variables type.
     // eslint-disable-next-line no-unused-vars
     TVariables: {...} = Empty,
-> = {|
+> = {
     type: TType,
     id: string,
-|};
+    // We allow other things here to be passed along to the fetch function.
+    // For example, we might want to pass the full query/mutation definition
+    // as a string here to allow that to be sent to an Apollo server that
+    // expects it.
+    ...
```

Comment:
What do you think about adding a test to cover this case?

Reply:
Well, we won't do anything with these values other than passing it all through so I don't know that it matters. There's not really anything to test.

Reply:
Fair enough :)